src/v2/Apps/BuyerGuarantee/Routes/BuyerGuaranteeIndex.tsx

```diff
@@ -58,13 +60,29 @@ export const BuyerGuaranteeIndex: React.FC<BuyerGuaranteeIndexProps> = ({
 
   const learnMoreIcon = (
     <Flex pt={2} justifyContent="center">
-      <Text variant="mediumText" mt="-2px">
-        Learn More
-      </Text>
-      <ArrowRightIcon height="15px" width="15px" />
+      <Link
+        href="https://support.artsy.net/hc/en-us/articles/360048946973"
```

Comment:
Potentially this can use the same variable below.

Reply:
+1 to @starsirius comment